plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -29,6 +30,7 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 	const [ showAddAttributeModal, setShowAddAttributeModal ] =
 		useState( false );
 	const onRemove = ( attribute: ProductAttribute ) => {
+		recordEvent( 'product_remove_attribute_button' );
```

Comment:
Suggestion: `product_remove_attribute_button_click`

Reply:
I removed this event in favor of the comment you made below.